pages/shared/syncHelpers.js

```diff
@@ -229,3 +233,57 @@ module.exports.gitStatus = function(locals, callback) {
         statusStage1();
     });
 };
+
+module.exports.ECRupdate = function(locals, callback) {
+    if (!config.externalGradingImageRepository) {
+        return callback(new Error('externalGradingImageRepository not defined'));
+    }
+
+    if (!AWS.config.credentials) {
+        return callback(new Error('AWS credentials not defined'));
+    }
+
+    dockerUtil.setupDockerAuth((err, auth) => {
+        if (ERR(err, callback)) return;
+
+        const options = {
+            course_id: locals.course.id,
+            user_id: locals.user.user_id,
+            authn_user_id: locals.authz_data.authn_user.user_id,
+            type: 'images_sync',
+            description: 'Sync Docker images from Docker Hub to PL registry',
+        };
+        serverJobs.createJobSequence(options, function(err, job_sequence_id) {
+            if (ERR(err, callback)) return;
+            callback(null, job_sequence_id);
+
+            var lastIndex = locals.images.length - 1;
+            async.eachOfSeries(locals.images || [], (image, index, done) => {
```

Comment:
```suggestion
            async.eachOfSeries(locals.images || [], (image, index, callback) => {
```

Prefer consistent use of `callback()` for callbacks.